### PR TITLE
[DRAFT][cluster-test] Implement an parallel option for cluster test benchmark

### PR DESCRIPTION
**Pull request by runtian-zhou**
## Motivation

Implemente a cluster test option for enabling parallel execution in benchmark suite.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

Yes
## Test Plan

TBD

### CODE REVIEW

File: diem-node/src/lib.rs [hunk 1]
```diff
@@ -139,8 +139,7 @@ pub fn load_test_environment<R>(
     template.api.address = template.json_rpc.address;
     template.json_rpc.stream_rpc.enabled = true;
     if lazy {
-        template.consensus.mempool_poll_count = 30;
-        template.consensus.safety_rules.network_timeout_ms = u64::MAX;
+        template.consensus.mempool_poll_count = u64::MAX;
```
Comment (zekun000) [inline diem-node/src/lib.rs:142]: not related?

**Review by zekun000 — COMMENTED** [review]
looks good to me, have you tried running the experiment?

File: diem-node/src/lib.rs (hunk 1 above)
Comment (runtian-zhou) [reply diem-node/src/lib.rs:142]: Sry updated!

**Review by zekun000 — CHANGES_REQUESTED** [review]
the experiment doesn't work 
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: No such file or directory '/diem/language/move-stdlib/sources'', language/move-stdlib/src/lib.rs:58:70
```

**Comment by runtian-zhou** [comment]
> the experiment doesn't work
> 
> ```
> thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: No such file or directory '/diem/language/move-stdlib/sources'', language/move-stdlib/src/lib.rs:58:70
> ```

Hmm must be realted with how move modules are stored. Lemme check

**Comment by zekun000** [comment]
I think the problem is that source code is not available in the built image, there's only binaries

**Comment by bmwill** [comment]
Could we instead add this to forge instead of cluster test since we're going to be turning down CT soon

**Comment by zihaoccc** [comment]
> Could we instead add this to forge instead of cluster test since we're going to be turning down CT soon

I have synced with @runtian-zhou about this point, this pr actually is a one time run job to get some perf numbers for proving of concept purpose, since their last baseline is adjusted by CT, another experiment written in Forge is also out and will be added soon into Forge

**Comment by zekun000** [comment]
we'd need cluster test to demonstrate 5k tps since the machines forge uses only have 4 cores

**Comment by bmwill** [comment]
Great if you're in the loop @zihaoccc then I'll get out of the way :)

**Comment by runtian-zhou** [comment]
Closing this PR as we will activate parallel execution benchmark in forge via a different genesis

**Pull request closed without merging**